iaac/terraform/aws-infra/subdomain/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "subdomain_zone_name" {
```

Comment:
@rrrkharse should this output be also named as `aws_route53_subdomain_zone_name`?

I kept different name to separate from input but it doesnt have to be the case, I think var and outputs can have same names

Reply:
var and outputs can have the same name

Reply:
ya, changed it to be consistent